ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec.json

```diff
@@ -368,7 +375,8 @@
     "CaseFieldID": "specRespondentSolicitor1EmailAddress",
     "UserRoles": [
       "caseworker-civil-solicitor",
-      "caseworker-civil-admin"
+      "caseworker-civil-admin",
+      "caseworker-civil-judge"
```

Comment:
Don't think a judge will require CRU access for specRespondentSolicitor1EmailAddress field?